examples/type_inference/type_inference_visitor_unittest.py

```diff
@@ -0,0 +1,243 @@
+"""Unittest for the type_inference_visitor."""
+
+import unittest
+import sys
+import astroid
+from typing import Tuple, List, Dict
+from astroid import test_utils
+from type_inference_visitor import TypeVisitor, set_const, set_binop, \
+    set_unaryop, set_dict, set_list, set_tuple
+
+
+class SetConstFunctionTest(unittest.TestCase):
+    """testers specifically for the function set_const using single nodes."""
+
+    def test_const_str(self):
+        """testing if the function set_const has the correct output for node
+        type str."""
+        expected = str
+        node = test_utils.extract_node("""a = __('sample_string')""")
```

Comment:
@hayleyZHL Okay, let me put it this way. I see no reason to use `extract_node` at all, instead of `astroid.parse` like the tests for all the other nodes.